lib/css/modules/_nightMode.scss

```diff
@@ -1148,7 +1148,8 @@
 	.snoovatar-page .gold-accent a,
 	.combined-search-page .search-result-subreddit .search-subscribe-button .add.active,
 	.combined-search-page .search-result-subreddit .search-subscribe-button .remove.active,
-	.fancy-toggle-button a {
+	.fancy-toggle-button a,
+	.fancy-toggle-button .multi-count {
```

Comment:
yeah, I always have night mode on so I noticed the difference immediately :smile: